src/offset/local/tz_info/timezone.rs

```diff
@@ -933,6 +936,338 @@ mod tests {
         Ok(())
     }
 
+    #[test]
+    #[allow(clippy::bool_assert_comparison)]
+    fn test_dst_backward_posix_tz() -> Result<(), Error> {
+        // Northern hemisphere DST (CET/CEST)
+        let tz: TimeZone = TimeZone::from_posix_tz("CET-1CEST,M3.5.0,M10.5.0/3")?;
```

Comment:
Nit: please drop all the type annotations for `tz` variables.

Reply:
Fair, I'll fix that.